Test/Radio Downloader Test.csproj

```diff
@@ -12,6 +12,8 @@
     <TargetFrameworkVersion>v2.0</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
This cruft that Visual Studio has added can be dropped out of this PR.

Reply:
With this and the comment below relating to 'Test/Radio Downloader Test.csproj', you suggest reverting or dropping out content.  It this best done via a git command or by direct editing the file through visual studio?

Reply:
When assembling the commit in the first place I often use `git add -p` to only add parts of a file.  However as the changes have already been committed I think you'd be best off hand editing them back out.